<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>authentication</class>
 <widget class="QDialog" name="authentication">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tracker authentication</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="login_logo">
       <property name="maximumSize">
        <size>
         <width>39</width>
         <height>39</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="login_title">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>39</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Tracker authentication</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="lbl_tracker">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Tracker:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="tracker_url">
       <property name="minimumSize">
        <size>
         <width>220</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Login</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="lbl_username">
          <property name="text">
           <string>Username:</string>
          </property>
          <property name="buddy">
           <cstring>lineUsername</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineUsername"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="lbl_passwd">
          <property name="minimumSize">
           <size>
            <width>68</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Password:</string>
          </property>
          <property name="buddy">
           <cstring>linePasswd</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="linePasswd">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="loginButton">
       <property name="text">
        <string>Log in</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>authentication</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>linePasswd</sender>
   <signal>returnPressed()</signal>
   <receiver>loginButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>122</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineUsername</sender>
   <signal>returnPressed()</signal>
   <receiver>linePasswd</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
